packages/react-aria-components/src/Breadcrumbs.tsx

```diff
@@ -16,11 +16,13 @@ import {filterDOMProps} from '@react-aria/utils';
 import {HeadingContext} from './Heading';
 import {LinkContext} from './Link';
 import {Node} from 'react-stately';
-import React, {createContext, ForwardedRef, forwardRef, HTMLAttributes} from 'react';
+import React, {createContext, ForwardedRef, forwardRef, HTMLAttributes, Key} from 'react';
 
 export interface BreadcrumbsProps<T> extends Omit<CollectionProps<T>, 'disabledKeys'>, Omit<AriaBreadcrumbsProps, 'children'>, StyleProps, SlotProps {
   /** Whether the breadcrumbs are disabled. */
-  isDisabled?: boolean
+  isDisabled?: boolean,
+  /** Handler that is called when a breadcrumb is clicked. */
+  onAction?: (key: Key) => void
```

Comment:
This matches spectrum's API, as a top level prop instead of needing to add onPress to every link yourself.